.github/workflows/build.yml

```diff
@@ -0,0 +1,51 @@
+name: kit
+on:
+  push: {}
+  release:
+    types: [published]
+  workflow_dispatch: {}
+permissions:
+  packages: write
+  contents: write
+  pull-requests: write
+  id-token: write
+env:
+  IMAGE_LOCALSTACK: docker.io/localstack/localstack:2.2.0@sha256:5468c261527460a907417887f2e3e5dc23c901f84c045919ff390bd755cf6f3e
+jobs:
+  go-build:
+    if: ${{ contains(fromJSON('["workflow_call", "push", "pull_request"]'), github.event_name) && startsWith(github.repository, 'GeoNet/') != false }}
+    uses: GeoNet/Actions/.github/workflows/reusable-go-build-smoke-test.yml@main
+  gofmt:
+    if: ${{ contains(fromJSON('["workflow_call", "push", "pull_request"]'), github.event_name) && startsWith(github.repository, 'GeoNet/') != false }}
+    uses: GeoNet/Actions/.github/workflows/reusable-gofmt.yml@main
+  golangci-lint:
+    if: ${{ contains(fromJSON('["workflow_call", "push", "pull_request"]'), github.event_name) && startsWith(github.repository, 'GeoNet/') != false }}
+    uses: GeoNet/Actions/.github/workflows/reusable-golangci-lint.yml@main
+  go-vet:
+    if: ${{ contains(fromJSON('["workflow_call", "push", "pull_request"]'), github.event_name) && startsWith(github.repository, 'GeoNet/') != false }}
+    uses: GeoNet/Actions/.github/workflows/reusable-go-vet.yml@main
+  go-test:
+    runs-on: ubuntu-latest
+    env:
+      AWS_REGION: ap-southeast-2
```

Comment:
note: shouldn't be necessary, as tests are written to set their own environment.

Reply:
@wilsonjord, ya reckon I should remove it?

Reply:
Originally didn't care, just noting. But on second thoughts, better to remove if possible, if only for the principle that it doesn't add anything of value.

Reply:
Removed in 7dab317